build.gradle.kts

```diff
@@ -11,5 +11,5 @@ buildscript {
 }
 
 plugins {
-    id("com.android.application") version "8.7.3" apply false
```

Comment:
Again, some changes that arn't required. You can rollback these.

Reply:
Okay @AsCress Thanks for Explaining me. I will do the needful !

Reply:
@AsCress Other than the SensorActivity the changes I have made because I was unable to run the application without making those changes. But after testing I will rollback it.

Reply:
@AsCress to resolve this issue as you have said, I need to make changes in the method "PopulateSensors" to handle the cases where both having same addresses.
should I ?

Reply:
@samruddhi-Rahegaonkar The problem here is with the data structure that we are using to store the sensor addresses.
 A `Map` in Java doesn't allow duplicate keys to be inserted. When we try to do that in lines 86 and 87 of `SensorActivity.java`, it overwrites the previous entries, and hence the previous two sensors (`MLX90614` and `TSL2561`) actually never get inserted in the `Map`. 
The idea here would be to migrate to a different data structure which would allow storage of duplicate entries, and then using that in `PopulateSensors` to display them.

Reply:
@AsCress Instead of `Map<Integer, String>`, we can use a `Map<Integer, List<String>>` to associate multiple sensor names with the same address.

Reply:
@samruddhi-Rahegaonkar Yep, that's one way. Go ahead and maybe try to implement whatever you find best in this PR.